spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -48,7 +48,7 @@ import org.apache.comet.shims.ShimQueryPlanSerde
  */
 object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
   def emitWarning(reason: String): Unit = {
-    logWarning(s"Comet native execution is disabled due to: $reason")
+    logWarning(s"Comet native execution: $reason")
```

Comment:
Hmm just wonder why is this better than the original?

Reply:
Oh I see (after reading the Github issue).

Reply:
Yea, we post the log because one certain expression/operator is not supported. Not all query is not supported. But currently the message looks like Comet native execution is disabled completely.